python/sdk/merlin/model.py

```diff
@@ -873,7 +874,6 @@ def log_pyfunc_model(self, model_instance, conda_env, code_dir=None,
             raise ValueError(
                 "log_pyfunc_model is only for PyFunc and PyFuncV2 model")
 
-        validate_model_dir(self._model.type, ModelType.PYFUNC, None)
```

Comment:
why removing this?

Reply:
Do we need this? I don't see any validation necessary for pyfunc model